mdantic.py

```diff
@@ -101,14 +103,13 @@ def mk_struct(cls, structs):
                 mk_struct(f.type_, structs)
 
 
-def fmt_tab(structs):
+def fmt_tab(structs, columns):
     tabs = {}
-    field_names = ["key", "type", "required", "description", "default"]
     for cls, struct in structs.items():
         tab = []
         for f in struct:
-            tab.append(list(f))
-        tabs[cls] = tabulate.tabulate(tab, headers=field_names, tablefmt="github")
+            tab.append([getattr(f,name) for name in columns])
```

Comment:
please add a space between parameters.

Reply:
Agreed and done. Should have run black just to check.